client/src/main/java/io/pravega/client/stream/Transaction.java

```diff
@@ -92,13 +92,16 @@
 
     /**
      * Drops the transaction, causing all events written to it to be deleted.
+     *
+     * @throws TxnFailedException Error aborting transaction: FAILURE
      */
-    void abort();
+    void abort() throws TxnFailedException;
 
     /**
      * Gets the status of the transaction.
      *
+     *  @throws TxnFailedException This exception could be thrown on checkStatus
```

Comment:
This comment does not add value. Can you please explain in which case, as an example, this method will throw this exception?

Reply:
Yes Enrico it does not throw TxnFailedException anymore.